examples/test/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"context"
+
+	"github.com/edaniels/golog"
+	"go.viam.com/rdk/robot/client"
+	"go.viam.com/rdk/utils"
+	"go.viam.com/utils/rpc"
+)
+
+func main() {
+	logger := golog.NewDevelopmentLogger("client")
+	robot, err := client.New(
+		context.Background(),
+		"ray-pi-main.tcz8zh8cf6.viam.cloud",
+		logger,
+		client.WithDialOptions(rpc.WithCredentials(rpc.Credentials{
+			Type:    utils.CredentialsTypeRobotLocationSecret,
+			Payload: "ewvmwn3qs6wqcrbnewwe1g231nvzlx5k5r5g34c31n6f7hs8",
+		})),
+	)
+	if err != nil {
+		logger.Fatal(err)
+	}
+	defer robot.Close(context.Background())
+	logger.Info("Resources:")
+	logger.Info(robot.ResourceNames())
+
```

Comment:
Oops I didn't mean to add the files in the `examples` folder.  Will remove

Reply:
I also recommend changing secrets on the pi, since this PR is public.

Reply:
...oops. Good catch